src/main.js

```diff
@@ -227,12 +228,30 @@ app.on('ready', function() {
 
     trayIcon.setToolTip(app.getName());
     trayIcon.on('click', function() {
+      if (process.platform === 'win32') {
+        if (config.minimizeToTray) {
+          if (mainWindow.isHidden) {
```

Comment:
I think they are temporary views to just know info. But this tray icon shows the main window of app. So the purpose is different between them.

Assuming your research, it seems that the behavior should be selectable. If possible, I hope the default behavior is "just show". Then, the entry in the context menu is a nice idea because users don't have to think about its configuration.